test/steps/v2_steps.py

```diff
@@ -2685,6 +2775,24 @@ def check_atomic_transaction_composer_response(context, returns):
             assert result.decode_error is None
 
 
+@then('The app should have returned ABI types "{abiTypes:MaybeString}".')
+def check_atomic_transaction_composer_return_type(context, abiTypes):
+    expected_tokens = abiTypes.split(":")
+    for i, expected in enumerate(expected_tokens):
```

Comment:
This should compare the lengths of `expected_tokens` and `context.atomic_transaction_composer_return.abi_results` as well

Reply:
👍